lib/barr/blocks/processes.rb

```diff
@@ -0,0 +1,15 @@
+module Barr
+  module Blocks
+
+    class Processes < Block
+      def initialize opts={}
+        super 
+        @output = sys_cmd
+      end
+
+      def sys_cmd
+        `ps -e | wc -l`
```

Comment:
Could you call `chomp` after the system cmd please? It's a minor detail as the manager will also remove any newlines as a safety measure, but I'd rather the blocks sanitised their own outputs as much as possible.